packages/extensions/ws/src/index.ts

```diff
@@ -298,14 +298,11 @@ class WebSocketExtension extends SdkExtension {
     if (Date.now() > this.wsTokenExpiresAt) {
```

Comment:
In the case that `wsToken` is mandatory. It's better to include the condition here when `wsToken` is null.

```js
if (!this.wsToken || Date.now() > this.wsTokenExpiresAt) {
```

Reply:
Need to consider on if it will cause a mass of API call to '/restapi/oauth/wstoken'.

Reply:
this.wsToken will never be undefined or null. Because the first time connect, `Date.now() > this.wsTokenExpiresAt` is always true and this.wsToken will be assigned a value. And the second/third/fourth... time connect,  `this.wsToken` already has a value

Reply:
I just added the logic suggested by you. It does no harm to add it.